proto/regen/ecocredit/basket/v1/query.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package regen.ecocredit.basket.v1;
+
+import "google/api/annotations.proto";
+import "regen/ecocredit/basket/v1/state.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket";
+
+// Msg is the regen.ecocredit.basket.v1beta1 Query service.
+service Query {
+
+    // Basket queries one basket by denom.
+    rpc Basket(QueryBasketRequest) returns (QueryBasketResponse) {
+        option (google.api.http).get =
+            "/regen/ecocredit/basket/v1/baskets/{basket_denom}";
+    }
+
+    // Baskets lists all baskets in the ecocredit module.
+    rpc Baskets(QueryBasketsRequest) returns (QueryBasketsResponse) {
+        option (google.api.http).get =
+            "/regen/ecocredit/basket/v1/baskets";
+    }
+
+    // BasketBalances lists the balance of each credit batch in the basket.
+    rpc BasketBalances(QueryBasketBalancesRequest) returns (QueryBasketBalancesResponse) {
+        option (google.api.http).get =
+            "/regen/ecocredit/basket/v1/baskets/{basket_denom}/balances";
+    }
+
+    // BasketBalanceByDenom queries the balance of a specific credit batch in the basket.
+    rpc BasketBalanceByDenom(QueryBasketBalanceByDenomRequest) returns (QueryBasketBalanceByDenomResponse) {
```

Comment:
why not just
```suggestion
    rpc BasketBalance(QueryBasketBalanceByDenomRequest) returns (QueryBasketBalanceByDenomResponse) {
```

Reply:
auto-merge disabled pending whether we change this

Reply:
ok changed it